src/cellfft/source/cellfft_get_kernel.cpp

```diff
@@ -0,0 +1,488 @@
+/*******************************************************************************
```

Comment:
Is there a reason to put this code to a separate module? If this is used only in one module (in cellfft solver, I guess), please put it into that module directly.

Reply:
I`ll put it into conv_cellfft.cpp directly